routing/world_graph.hpp

```diff
@@ -39,7 +39,7 @@ class WorldGraph
 
   // |isEnding| == true iff |segment| is first or last segment of the route. Needed because first and
   // last segments may need special processing.
-  virtual void GetEdgeList(Segment const & segment, bool isOutgoing, bool isLeap, bool isEnding,
+  virtual void GetEdgeList(Segment const & segment, bool isOutgoing, bool isLeap,
```

Comment:
If you remove the |isEnding| param pls fix the comment.

Reply:
done